src/testStatusBarProvider.ts

```diff
@@ -40,6 +40,9 @@ class TestStatusBarProvider implements Disposable {
             }
         }
 
+        this.statusBarItem.accessibilityInformation = {
+            label: `${failedNum} failed, ${passedNum} passed test${failedNum + passedNum > 1 ? 's' : ''}`,
```

Comment:
AFAIK, 0 is plural in English, maybe `num === 1 ? '' : "s" ` would be better?

Reply:
yes that's better!